test/operations-conditions-1.js

```diff
@@ -0,0 +1,12 @@
+var assert = require("assert");
```

Comment:
no need to have require in files itself, it will be enough to add it in test file